RenderStateNotation/src/RenderStateNotationParserImpl.cpp

```diff
@@ -260,17 +260,23 @@ Bool RenderStateNotationParserImpl::ParseFile(const Char* FileName, IShaderSourc
 
     try
     {
-        RefCntAutoPtr<IFileStream> pFileStream;
-        pStreamFactory->CreateInputStream(FileName, &pFileStream);
+        if (m_Includes.find(FileName) == m_Includes.end())
```

Comment:
You can use `if(m_Includes.find(FileName).second)` to save one `insert`

Reply:
This isn't a map

Reply:
[std::unordered_set::insert](https://en.cppreference.com/w/cpp/container/unordered_set/insert) works similar to std::unordered_map::insert